frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -23,7 +23,21 @@ type MetabaseComponentThemeKey = FlattenObjectKeys<MetabaseComponentTheme>;
 export function getMetabaseCssVariables(theme: MantineTheme) {
   return css`
     :root {
-      ${getDesignSystemCssVariables(theme)}
+      --mb-default-font-family: "${theme.fontFamily}";
+
+      /* Semantic colors */
+      --mb-color-brand: ${theme.fn.themeColor("brand")};
+      --mb-color-summarize: ${theme.fn.themeColor("summarize")};
+      --mb-color-filter: ${theme.fn.themeColor("filter")};
```

Comment:
we only use these 3 colors dynamically in the main app via CSS custom properties, other colors `accentx` aren't referenced from CSS custom properties, so we don't need to define them here.